src/OpenTelemetry.Extensions/Logs/OpenTelemetryLoggingExtensions.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="OpenTelemetryLoggingExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET461_OR_GREATER || NETSTANDARD2_0 || NET5_0_OR_GREATER
+using System;
+using System.Diagnostics;
+using OpenTelemetry.Logs;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Contains OpenTelemetry logging SDK extensions.
+    /// </summary>
+    public static class OpenTelemetryLoggingExtensions
+    {
+        /// <summary>
+        /// Adds a LogRecord Processor to the OpenTelemetry ILoggingBuilder
+        /// which converts messages into <see cref="ActivityEvent"/>s on the
+        /// currently active <see cref="Activity"/>.
+        /// </summary>
+        /// <param name="loggerOptions"><see cref="OpenTelemetryLoggerOptions"/> options to use.</param>
+        /// <param name="configure"><see cref="ActivityEventAttachingLogProcessorOptions"/>.</param>
+        /// <returns>The instance of <see cref="OpenTelemetryLoggerOptions"/> to chain the calls.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="loggerOptions"/> is <c>null</c>.</exception>
+        public static OpenTelemetryLoggerOptions AddActivityEventAttachingLogProcessor(
```

Comment:
nit. The fact that logs get attached as activity event, via a LogProcessor is an implementation detail, right? An alternate signature:
`public static OpenTelemetryLoggerOptions AttachLogsToActivityEvent(this OpenTelemetryLoggerOptions loggerOptions,
Action<ActivityTagsCollection, IReadOnlyList<KeyValuePair<string, object>>> stateConvertor = DefaultLogStateConverter.ConvertState,
Action<ActivityTagsCollection, int, LogRecordScope> scopeConverter = DefaultLogStateConverter.ConvertScope)`

Reply:
Updated the extension name over on contrib. I also renamed the options to `LogToActivityEventConversionOptions` which seemed a little more friendly than `ActivityEventAttachingLogProcessorOptions`. I felt like what you _really_ wanted to do here was eliminate "ActivityEventAttachingLogProcessor" anywhere in the public API, which the rename achieved. I like having the options class more than this signature but LMK we can continue the discussion over on the new PR if needed.